content/blog/2025-06-15-cancellation.md

```diff
@@ -0,0 +1,353 @@
+---
```

Comment:
I took the liberty of pushing this "front matter" and ASF header to this post 

I tried to [render the blog locally](https://github.com/apache/datafusion-site?tab=readme-ov-file#setup-for-docker) to preview but found it wasn't in the blog list without this

![Screenshot 2025-06-17 at 12 49 31 PM](https://github.com/user-attachments/assets/a1bfbc5b-11fc-4ce4-83e4-b94a6acab0f0)

Reply:
Since Datadobi is paying for my time while I work on this stuff, would it be possible to add a small note mentioning that?

Reply:
Yes of course -- we have done this on other blogs as well 

Something like "Thank you to Datadobi for supporting this work" would be very much following the same pattern